app/src/main/java/com/ddoczi/tasky/util/DispatcherProviderImpl.kt

```diff
@@ -0,0 +1,11 @@
+package com.ddoczi.tasky.util
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+
+class DispatcherProviderImpl : DispatcherProvider {
```

Comment:
If a class doesn't need a constructor and doesn't have internal state, you can make it an `object` instead :)